core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/TagsImpl.java

```diff
@@ -66,6 +64,11 @@
     private Page currentPage;
 
     private List<String> tagTitles;
+    
+    @ValueMapValue
+    @Optional
+    @Default(values = "false")
+    private Boolean displayComputedProperty;
```

Comment:
I don't understand why this flag is needed?

Reply:
This could be taken out, initially we wanted to add it for retaining the existing functionality.